src/api/app/models/comment.rb

```diff
@@ -3,6 +3,7 @@
 class Comment < ApplicationRecord
   belongs_to :commentable, polymorphic: true # belongs to a Project, Package, BsRequest or BsRequestActionSubmit
   belongs_to :user, inverse_of: :comments
+  belongs_to :moderator, class_name: 'User', optional: true
```

Comment:
Hm, why is this optional?

Reply:
Because it's not compulsory to always have a moderator associated with the comment.

Reply:
Double-checking... 

Indeed. If I remove `optional: true` and try to create a comment without specifying a moderator I get:

```Validation failed: Moderator must exist
```